### PR TITLE
AWSAccountID UI component

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

File: src/components/UI/theme.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ const theme = {
     med: '825px',
     large: '1200px',
   },
+  fontFamilies: {
+    console: 'Inconsolata, monospace',
+  },
```
Comment (marians) [inline src/components/UI/theme.js]: As I understand it, the naming here is up to us, right?

Comment (ograu) [reply src/components/UI/theme.js]: Yes, we can use the naming we want

**Comment by marians** [comment]
@ograu I did that, didn't I? Have I forgotten anything? (See last file in https://github.com/giantswarm/happa/pull/610/files)

**Comment by ograu** [comment]
You are so right! Sorry...

**Pull request merged**